setup.cfg

```diff
@@ -67,6 +67,8 @@ filterwarnings =
     ignore:leap-second auto-update failed:astropy.utils.exceptions.AstropyWarning
 # Should ignore these for astropy<5.0
     ignore:getName|currentThread:DeprecationWarning:astropy
+# Ignore errors from astropy.wcs about the number of WCS axes not matching image dimension
+    ignore:The WCS transformation has more axes:astropy.wcs.wcs.FITSFixedWarning
```

Comment:
This is a very strange issue in CI - the easiest way to get around it is to not specify the warning class here. It seems to occur for all Python versions except Python 3.11 so not sure what is going on there. The issue doesn't seem to happen if instead you put:

```
@pytest.mark.filterwarnings("ignore:The WCS transformation has more axes:astropy.wcs.wcs.FITSFixedWarning")
```

on the relevant tests. I think it makes sense anyway to filter it at the test level as it's not really the kind of warning we usually filter in ``setup.cfg`` which are more things like deprecations and so on that are beyond our control. The above warning is a real WCS warning that will always occur and is expected.

Reply:
I would have opened a PR to your branch @mwcraig but I couldn't figure out which test was actually emitting this warning (couldn't get the astrometry remote tests to run properly)

Reply:
Yes, that would be indeed nice to filter this on the test level, thanks Tom for suggesting it.

Reply:
Thanks for the suggestion -- I can make the change tomorrow morning when I'm back at home.

Reply:
The fix has been made, hopefully the CI passes now (it is passing locally).